spec/controllers/articles_controller_spec.rb

```diff
@@ -27,20 +27,21 @@
         end
     end
 
-    describe '#update' do
-      describe 'update a record that already exists,'do
+    describe 'GET #update' do
+      describe 'update a record that already exists'do
         context 'when it finds Article' do
           it 'redirect to article' do
+            expect(subject).to redirect_to(@article)
           end
         end
         context 'when not it find article' do
-          it 'redirect to new article page' do
+          it 'create a new article' do
           end
         end
       end
     end
 
-    describe '#destroy' do
+    describe 'GET #destroy' do
```

Comment:
This is not the correct HTTP method. Please refer to this: http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions.